CHANGELOG

```diff
@@ -1,11 +1,14 @@
 v3.7.3 (2020-10-14)
 -------------------
-      
+
+* Fix a memory leak because cluster internal connections were not cleaned
+  up for agency communication.
```

Comment:
This adds the new option to the CHANGELOG of the already-released version 3.7.3.
I guess the CHANGELOG entry should go into the section for version 3.7.5 instead.

Reply:
3.7.3.1 will have it's own section.